### PR TITLE
VocSens added as new company.

### PR DESCRIPTION
**Pull request by njulemon**
#### Summary
Added VocSens company & new device msem-10.

#### Changes
Added VocSens company & new device msem-10.

#### Notes for Reviewers
Ben Olayinka has been in touch with Bram Senave to add VocSens to device repo. 

#### Release Notes
First Time v-0.1

### CODE REVIEW

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @njulemon files need to be filled `index.yaml`  and  `msem-10.yaml`

**Comment by Jaime-Trinidad** [comment]
@njulemon any update here?

**Comment by Jaime-Trinidad** [comment]
Hi @njulemon please complete the files, so we can merge the devices. Thanks

**Comment by Jaime-Trinidad** [comment]
Hi @njulemon any update?

File: vendor/vocsens/msem-10-profile-eu868.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+The Device Repository supports three payload codecs to be defined:
+
+Uplink decoder: decodes binary data uplink into a JSON object
+Downlink encoder: decodes a JSON object into binary data downlink
+Downlink decoder: decodes an encoded binary data downlink back into a JSON object (must be symmetric with the downlink encoder)
+Note: We recommend using measure units and full variable names in payload decoders for a better user experience.
+
+The codecs can all be defined in one file as they are defined by their function names. The codecs must be written in JavaScript (ECMAScript 5.1+). See link for instructions on how to write decoders and encoders.
+
+The codecs are defined in the Payload codec definition file, with the same filename as the codec ID: vendor/<vendor-id>/<codec>.yaml:
```
Comment (johanstokking) [inline vendor/vocsens/msem-10-profile-eu868.yaml:65]: This should be commented, although it looks technically like valid YAML

**Review by johanstokking — CHANGES_REQUESTED** [review]
- Please add `vocsens` to `vendor/index.yaml`
- Please remove empty `vendor/vocsens/msem-10.yaml`
- Please address review comments

**Comment by johanstokking** [comment]
Ah it looks like everything is combined in one file. It should be split over end device, codec and profile files. See `vendor/example`.

**Comment by Jaime-Trinidad** [comment]
Hi @njulemon I will close this PR due time.

**Pull request closed without merging**